.github/workflows/stale.yaml

```diff
@@ -0,0 +1,47 @@
+name: Mark and Close Stale Issues and PRs
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # runs every day at midnight UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Mark and close stale issues and PRs
+        uses: actions/stale@v9
+        with:
+          remove-stale-when-updated: true
+          stale-issue-label: 'lifecycle/stale'
+          exempt-issue-labels: 'kind/feature, kind/bug, help wanted, good first issue'
```

Comment:
It is recommended to remove these labels. Keep `exempt-issue-labels` empty

Reply:
done

Reply:
I think exempt-issue-labels can help us avoid some valuable issues being marked as stale, which will make it difficult to find and fix them later @hwdef .

Reply:
> I think exempt-issue-labels can help us avoid some valuable issues being marked as stale, which will make it difficult to find and fix them later @hwdef .

Yes, but we did not have a  norm.Most of the issues have `kind/feature, kind/bug, help wanted, good first issue`.So the stale will be useless.

Reply:
Sorry, I missed https://github.com/volcano-sh/volcano/pull/4266#discussion_r2084147226

Please follow this comment, forget what I say.

Reply:
> > I think exempt-issue-labels can help us avoid some valuable issues being marked as stale, which will make it difficult to find and fix them later @hwdef .
> 
> Yes, but we did not have a norm.Most of the issues have `kind/feature, kind/bug, help wanted, good first issue`.So the stale will be useless.

@hwdef Yes, basically all of issues. I excluded time-sensitive ones like `kind/question`. I think `good first issue` and `help wanted` are more valuable. `kind/bug` is also user feedback that needs to be fixed in time, but I am not sure about `kind/feature`, or if you have other opinions,

Reply:
@JesseStutler 
You are right.

Reply:
Maybe we can keep "good first issue" and "help wanted" and remove "kind/feature", "kind/bug"? @Monokaix @hwdef How are other repositories defined, such as Kubernetes test-infra https://github.com/kubernetes/test-infra? @mahdikhashan

Reply:
Maybe we can keep "good first issue" and "help wanted" and remove "kind/feature", "kind/bug"? @Monokaix @hwdef How are other repositories defined, such as Kubernetes test-infra https://github.com/kubernetes/test-infra? @mahdikhashan

Reply:
I think it's better to keep all of them.